tap_s3_csv/s3.py

```diff
@@ -99,6 +99,7 @@ def merge_dicts(first, second):
 
 def sample_file(config, table_spec, s3_path, sample_rate):
     file_handle = get_file_handle(config, s3_path)
+    # _raw_stream seems like the wrong way to access this..
```

Comment:
can you please give a little more details and suggestions about it?

Reply:
it appears to be a private property, which means there is probably a better way to iterate over the file.  i don't have a suggestion.